awscli/examples/rum/delete-rum-metrics-destination.rst

```diff
@@ -0,0 +1,10 @@
+**To deletes a destination for CloudWatch RUM extended metrics**
+
+The following ``delete-rum-metrics-destination`` deletes a destination for CloudWatch RUM extended metrics. If the command succeeds, no output is returned. ::
+
+	aws rum delete-rum-metrics-destination \
```

Comment:
Same here. All of these indents need to be 4 spaces each and not tab indenting.
For more information: https://aws.github.io/aws-cli/docs_styleguide.html#general